src/HtmlElement.php

```diff
@@ -122,7 +122,7 @@ public function __toString()
             $result .= ' ' . $key . '="' . $value . '"';
         }
 
-        if ($this->contents) {
+        if (!is_null($this->contents) and $this->contents != '') {
```

Comment:
Yep, that was my bad: https://github.com/thephpleague/commonmark/commit/34884ce26b1a571d46c5d8ff7c6bc172e45aea13